examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_3.ipynb

```diff
@@ -0,0 +1,139 @@
+    "\n",
+    "**GOAL**: Learn what happens when a function is not yet supported in Modin."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When functionality has not yet been implemented, we default to pandas\n",
+    "\n",
+    "![](../../../img/omnisci_convert_to_pandas.png)\n",
+    "\n",
+    "We convert the Modin dataframe to a pyarrow.Table, perform a lazy tree in OmniSci, render it as a pyarrow.Table, convert it to pandas to perform the operation, and then convert it back to Modin when complete. These operations will have a large overhead due to the communication involved and will take longer than pandas.\n",
+    "\n",
+    "When this is happening, a warning will be given to the user to inform them that this operation will take longer than usual. For example, `DataFrame.mask` is not yet implemented. In this case, when a user tries to use it, they will see this warning:\n",
+    "\n",
+    "```\n",
+    "UserWarning: `DataFrame.mask` defaulting to pandas implementation.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concept for exercise: Default to pandas\n",
+    "\n",
+    "In this section of the exercise we will see first-hand how the runtime is affected by operations that are not implemented."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import modin.pandas as pd\n",
+    "import pandas\n",
+    "import numpy as np\n",
+    "import time\n",
+    "import modin.config as cfg\n",
+    "cfg.StorageFormat.put(\"omnisci\")\n",
+    "\n",
+    "frame_data = np.random.randint(0, 100, size=(2**18, 2**8))\n",
+    "df = pd.DataFrame(frame_data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas_df = pandas.DataFrame(frame_data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "modin_start = time.time()\n",
+    "\n",
+    "print(df.apply(sum))\n",
+    "\n",
+    "modin_end = time.time()\n",
+    "print(\"Modin mask took {} seconds.\".format(round(modin_end - modin_start, 4)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas_start = time.time()\n",
+    "\n",
+    "print(pandas_df.apply(sum))\n",
+    "\n",
+    "pandas_end = time.time()\n",
+    "print(\"pandas mask took {} seconds.\".format(round(pandas_end - pandas_start, 4)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## To request a feature please open an issue: https://github.com/modin-project/modin/issues\n",
+    "\n",
+    "For a complete list of what is implemented, see the [documentation](https://modin.readthedocs.io/en/latest/UsingPandasonRay/dataframe_supported.html)"
```

Comment:
```suggestion
    "For a complete list of what is implemented, see the [Supported APIs](https://modin.readthedocs.io/en/latest/supported_apis/index.html) section."
```

Reply:
Could you fix this link in other notebooks for pandas_on_ray/dask as part of a separate PR?

Reply:
@YarShev Yes, I could

Reply:
Create an issue for this please.

Reply:
@YarShev https://github.com/modin-project/modin/issues/4280

Reply:
@YarShev @YarShev I changed this lines in this tutorial.